bin/mqtt5_canary/main.cpp

```diff
@@ -796,58 +852,65 @@ int main(int argc, char **argv)
     {
         struct AwsMqtt5CanaryTestClient client;
         client = {};
+        client.clientId = String("TestClient") + std::to_string(i).c_str();
```

Comment:
Nitpick: Should a UUID be appended here so it cannot conflict with another run of the canary?